docs/advanced_topics/performance.md

```diff
@@ -92,6 +92,80 @@ For some images, it may be beneficial to lazy load images, so the rest of the pa
 
 This optimisation is already handled for you for images in the admin site.
 
+## Template fragment caching
+
+Django supports [template fragment caching](https://docs.djangoproject.com/en/stable/topics/cache/#template-fragment-caching), which allows caching portions of a template. Using Django's `{% cache %}` tag natively with Wagtail can be [dangerous](https://github.com/wagtail/wagtail/issues/5074) as it can result in preview content being shown to end users. Instead, Wagtail provides 2 extra template tags which can be loaded from `wagtail_cache`:
+
+### Preview-aware caching
+
+`{% wagtailcache %}` functions identically to Django's `{% cache %}` tag, but does nothing during preview requests (it neither reads from the cache nor updates it when one doesn't exist).
```

Comment:
```suggestion
The `{% wagtailcache %}` tag functions similarly to Django's `{% cache %}` tag, but will neither cache or serve cached content when previewing a page (or other model) in Wagtail.
```

Reply:
A couple of very small things, really;
— 'identically' didn't feel quite right to me when immediately followed by a description of a thing that is different
— The 'does nothing' description didn't feel very useful, but the stuff in brackets did, so thought we might as well just explain the difference in a single sentence for better 'flow' .